website/www/site/layouts/index.html

```diff
@@ -0,0 +1,149 @@
+{{/* 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License. See accompanying LICENSE file.
+*/}}
+
+{{ define "hero-section" }}
+  <div class="hero-bg">
+    <div class="hero section">
+      <div class="hero__cols">
+        <div class="hero__cols__col">
+          <div class="hero__cols__col__content">
+            <div class="hero__title">
+              Apache Beam: An advanced unified programming model
+            </div>
+            <div class="hero__subtitle">
+              Implement batch and streaming data processing jobs that run on any execution engine.
+            </div>
+            <div class="hero__ctas hero__ctas--first">
+              <a class="button button--primary" href="">Learn more</a>
+              <a class="button button--primary" href="">Try Beam</a>
+              <a class="button button--primary" href="">Download Beam SDK {{ .Site.Params.release_latest }}</a>
+            </div>
+            <div class="hero__ctas">
+              <a class="button" href="">Java Quickstart</a>
+              <a class="button" href="">Python Quickstart</a>
+              <a class="button" href="">Go Quickstart</a>
+            </div>
+          </div>
+        </div>
+        <div class="hero__cols__col">
+          <div class="hero__blog">
+            <div class="hero__blog__title">
+              The latest from the blog
+            </div>
+            <div class="hero__blog__cards">
+              {{/* TODO: Depends on BLOG page
```

Comment:
This is a loop for the three first blogs on the Blog page. I comment it because we don't have any blog at the moment.

Reply:
sure 👍